### PR TITLE
Feature/create.ts cmd type

### PR DESCRIPTION
**Pull request by ATorrise**
changed parameter from `cmdType` to `dataSetType` and clarified options description

### CODE REVIEW

File: packages/zosfiles/src/methods/create/Create.ts [hunk 1]
```diff
@@ -34,13 +34,13 @@ export class Create {
     /**
      * Create a data set
      * @param {AbstractSession} session                     - z/OSMF connection info
-     * @param {CreateDataSetTypeEnum} cmdType               - The type of data set we are going to create
+     * @param {CreateDataSetTypeEnum} dataSetType           - The type of data set we are going to create from default options
      * @param {string} dataSetName                          - the name of the data set to create
      * @param {Partial<ICreateDataSetOptions>} [options={}] - additional options for the creation of the data set
```
Comment (t1m0thyj) [inline packages/zosfiles/src/methods/create/Create.ts:39]: Should these docstrings be updated based on the changes we discussed?

File: packages/zosfiles/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
+## Recent Changes
+- Enhancement: Added to the description of Create.ts `cmdType`. [#48](https://github.com/zowe/zowe-cli/issues/48)
```
Comment (gejohnston) [inline packages/zosfiles/CHANGELOG.md:6]: I suggest rewording the changelog text. We anticipate that our users will read the changlog to know if there are changes of interest to them. The reference to a source file is technically correct, but will not mean much to our users. Try stating the change in terms of the command that has been affected and the result that the end user will see. It still does not have to be very wordy - just a different perspective.

**Review by gejohnston — CHANGES_REQUESTED** [review]
One minor suggestion about the wording of some text.

**Review by t1m0thyj — APPROVED** [review]

**Review by gejohnston — APPROVED** [review]

File: packages/zosfiles/CHANGELOG.md (hunk 2 above)
Comment (zFernand0) [reply packages/zosfiles/CHANGELOG.md:6]: Should this be considered a bugFix / patch since it's not introducing new behavior

**Review by zFernand0 — APPROVED** [review]
Thanks for fixing the JSDoc!
left one small comment (which can be address in other PRs 😋
